backup_and_restore/application_backup_and_restore/release-notes/oadp-1-4-release-notes.adoc

```diff
@@ -0,0 +1,29 @@
+:_mod-docs-content-type: ASSEMBLY
+
+[id="oadp-1-4-release-notes"]
+= OADP 1.4 release notes
+
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: oadp-1-4-release-notes
+
+toc::[]
```

Comment:
The TOC is not rendering. I recommend formatting this page exactly like the 1.3 release notes, see if that fixes the problem: 
```
:_mod-docs-content-type: ASSEMBLY
[id="oadp-release-notes"]
= OADP 1.3 release notes
include::_attributes/common-attributes.adoc[]
include::_attributes/attributes-openshift-dedicated.adoc[]
:context: oadp-release-notes

toc::[]

Reply:
fixed